src/main/java/lotto/model/LottoNumber.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.Objects;
 
-public class LottoNumber {
+public class LottoNumber implements Comparable<LottoNumber> {
     private final int number;
 
```

Comment:
```suggestion
public LottoNumber(String number)
```
문자열을 받는 생성자를 추가할 경우 의미있을까?